gobblin-temporal/src/main/java/org/apache/gobblin/temporal/cluster/GobblinTemporalClusterManager.java

```diff
@@ -152,7 +141,7 @@ private void initializeAppLauncherAndServices() throws Exception {
     if (!properties.contains(ServiceBasedAppLauncher.APP_STOP_TIME_SECONDS)) {
       properties.setProperty(ServiceBasedAppLauncher.APP_STOP_TIME_SECONDS, Long.toString(300));
     }
-    this.applicationLauncher = new ServiceBasedAppLauncher(properties, this.clusterName);
+    this.applicationLauncher = new ServiceBasedAppLauncherWithoutMetrics(properties, this.clusterName);
```

Comment:
Metrics in these pipelines are causing noisy logging and the metrics here are currently not used by Temporal this temporal in any meaningful way, so I've disabled them here.

Disabling metrics via the gobblin metris key is not desirable because we still need GTE emission to work